tests/ingestion_tm/test_ingest.py

```diff
@@ -0,0 +1,40 @@
+import inspect
```

Comment:
small test file, but it makes sure that when we're adding new TMExport classes, we remember to add them to the pipeline.